### PR TITLE
ci: add wf for vpnx release

### PR DESCRIPTION
**Pull request by doums**
- add a wf for vpnx release (new desktop client)
- only support linux and macos
- based on the current production release wf

### CODE REVIEW

File: .github/workflows/release-nym-vpn-x.yml [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+        run: |
+          ls -la ./
+          rm -rf ./* || true
+          rm -rf ./.??* || true
+          ls -la ./
+
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
```
Comment (octol) [inline .github/workflows/release-nym-vpn-x.yml:55]: In the past this has bitten us were jobs reused old files, so imho let's skip sparse checkout

**Review by octol — CHANGES_REQUESTED** [review]

File: .github/workflows/release-nym-vpn-x.yml (hunk 1 above)
Comment (doums) [reply .github/workflows/release-nym-vpn-x.yml:55]: these are the only nym deps
image

so it's fine (should be)

Comment (octol) [reply .github/workflows/release-nym-vpn-x.yml:55]: Well, should be, but there is also no performance advantage to doing a sparse checkout at least for our repo. So imho we should take the safe(r) route here

**Comment by doums** [comment]
Do we need different secrets for tauri signing and apple singing stuff?

File: .github/workflows/release-nym-vpn-x.yml (hunk 1 above)
Comment (doums) [reply .github/workflows/release-nym-vpn-x.yml:55]: ok forgot "should be", it's just fine

Comment (octol) [reply .github/workflows/release-nym-vpn-x.yml:55]: imho let's remove the sparse please

**Review by zaneschepke — APPROVED** [review]

**Pull request closed without merging**